Library/Formula/git-annex.rb

```diff
@@ -14,33 +14,24 @@ class GitAnnex < Formula
     sha1 "bf687c017b26ac5694e466afd8fe996c7ea2c22b" => :mountain_lion
   end
 
-  depends_on "gcc" => :build
   depends_on "ghc" => :build
   depends_on "cabal-install" => :build
   depends_on "pkg-config" => :build
-  # wget is workaround for http://git-annex.branchable.com/bugs/Build_fails_when_no_wget_avalible/
-  depends_on "wget" => :build
   depends_on "gsasl"
   depends_on "libidn"
   depends_on "gnutls"
-  depends_on "gmp"
   depends_on "quvi"
 
-  fails_with(:clang) { build 425 } # clang segfaults on Lion
-
   def install
     cabal_sandbox do
       cabal_install_tools "alex", "happy", "c2hs"
-      # gcc required to build gnuidn
-      gcc = Formula["gcc"]
-      cabal_install "--with-gcc=#{gcc.bin}/gcc-#{gcc.version_suffix}",
-                    "--only-dependencies",
-                    "--constraint=utf8-string==0.3.8" # use older utf8-string until 'feed' is updated
+      cabal_install "--only-dependencies"
```

Comment:
This should probably reported/fixed upstream at https://git-annex.branchable.com/bugs and/or https://github.com/bos/bloomfilter, but if you add `"--constraint=bloomfilter<2.0.0.0"` here, I believe it will build.